.github/workflows/spice-qa.yml

```diff
@@ -78,8 +65,17 @@ jobs:
             exit 1
           fi
 
+      - name: Retrieve response from streaming result
+        shell: bash
+        env:
+          SPICE_API_KEY: ${{ secrets.SPICE_API_KEY }}
+        run: |
+          input_file="api_responses/${{ env.RECIPE_FILE_PATH }}_stream.log"
```

Comment:
If we need to write external code to do this, we should write it in the spice CLI, not make another program.